test/0_staking.js

```diff
@@ -88,4 +90,16 @@ describe('Candidates make stakes on themselves', () => {
             expect(fStakeBN, `Stake on candidate ${candidate.staking} didn't increase`).to.be.bignumber.equal(iStakeBN.add(stakeBN));
         }
     });
+
+    it('Candidates are in validator set in the new staking epoch', async() => {
+        console.log('***** Wait for staking epoch to change');
+        await waitForValidatorSetChange(web3);
```

Comment:
Do you really need to wait for 2 staking epochs?

Reply:
:+1: